etc/curator-2-check.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+set -euxo pipefail
+
+mkdir -p ~/.local/bin
+export PATH=$HOME/.local/bin:$PATH
+
+# Get new Stackage curator
+CURATOR2=stackage-curator-2-9132b4b184f1f7f27c4c80ec0609460cea22ae75
+wget  "https://s3.amazonaws.com/www.snoyman.com/stackage-curator-2/$CURATOR2.bz2"
+bunzip2 "$CURATOR2.bz2"
+chmod +x $CURATOR2
+mv $CURATOR2 ~/.local/bin/stackage-curator-2
+
+# New curator check
+stackage-curator-2 update &&
+  stackage-curator-2 constraints &&
+  stackage-curator-2 snapshotincomplete &&
+  stackage-curator-2 snapshot &&
+  stackage-curator-2 checksnapshot
```

Comment:
This is more a comment on the new curator tool than on this PR, but can there be a convenience command `stackage-curator-2 check` that performs all of these steps? (If this is not desired, I can just make an `alias` locally, but it would be nice to keep curators on the same page with an easy user experience.)

Reply:
FWIW I always use the `check` script in this repo already, and we could update it to work with the new curator tool. We could even go farther and have it automatically install the tool based on a Git commit.

Reply:
+1 you're right; just having the `check` script is fine.
Auto-install is an interesting idea. My initial reaction is skepticism that this is a thing that I would want, but I could be convinced.

Reply:
I'm on OS X so I'm guessing this won't work for me? Should i install it from the stack repo?

Reply:
Yes, it's in https://github.com/commercialhaskell/stack/tree/master/subs/curator